cwatch.php

```diff
@@ -949,6 +949,21 @@ private function getServiceInfo($template, $service)
         $this->view->set('licenses', $licenses);
         $this->log('viewinfo', serialize($licenses), 'output', true);
 
+        $login_url = 'http://cww-customerportal-stackpath-takeover.us-east-1.elasticbeanstalk.com/';
```

Comment:
A log in URL shouldn't ever be over insecure HTTP.

Is this URL the preferred URL to log in to? Specifically one of the US East servers? No generic log in URL (e.g. load balancer) that will automatically redirect to the appropriate log in portal?